src/directives/decorators/bootstrap/bootstrap-decorator.js

```diff
@@ -9,6 +9,7 @@ angular.module('schemaForm').config(['schemaFormDecoratorsProvider', function(de
       console.log('fieldset children frag', children.childNodes)
       args.fieldFrag.childNode.appendChild(children);
     }},*/
+    formselect: {template: base + 'formselect.html', replace: false},
```

Comment:
This has now moved to https://github.com/json-schema-form/angular-schema-form-bootstrap